draft-ietf-quic-qpack.md

```diff
@@ -1130,8 +1141,8 @@ The following error codes are defined for HTTP/3 to indicate failures of
 QPACK which prevent the connection from continuing:
 
 QPACK_DECOMPRESSION_FAILED (0x200):
-: The decoder failed to interpret a header block and is not able to continue
-  decoding that header block.
+: The decoder failed to interpret a representation and is not able to continue
```

Comment:
Consider writing "an encoded field section" instead of "a representation" to include errors interpreting data in prefix.